File: Exemplos/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/BRIDGE/Commands/ImprimirCupomSat.ts
import { BridgeCommand } from "../BridgeCommand";

export class ImprimirCupomSat extends BridgeCommand {
  readonly xml: string;

  constructor(xml: string) {
    super("ImprimirCupomSat");
    this.xml = xml;
  }

  functionParameters(): object {
    return { xml: this.xml };
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid-font/IntentServices/Bridge/ImprimirCupomSat.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Bridge
{
    class ImprimirCupomSat : BridgeCommand
    {
        readonly string xml;

        public ImprimirCupomSat(string xml) : base("ImprimirCupomSat")
        {
            this.xml = xml;
        }

        protected override string FunctionParameters()
        {
            return "\"xml\"" + ":" + "\"" + xml + "\"";
        }
    }
}